test/windows/plugin.spec.ts

```diff
@@ -78,8 +78,8 @@ describe("windows scanning", () => {
     ]);
   });
 
-  it("can static scan for Identifier type image (nginx:1.19.0)", async () => {
-    const imageNameAndTag = "nginx:1.19.0";
```

Comment:
ngnix:1.19.0 is expected to fail here but the image does exist; changing to version 1.19.11 that does not exist and never will

Reply:
So how did this test pass until now? 🤔

Reply:
That is something I am unsure of too, it should not of passed. The test is clearly to check that it fails when the image does not exist, this test looks like it was written before 1.19.0 existed but it should have been failing for a while now. I would like to understand the logic of why it was not failing previously but for now this seems like a valid change.